omnibus/config/software/datadog-agent.rb

```diff
@@ -144,7 +144,8 @@
   end
 
   # System-probe
-  sysprobe_support = (not heroku_target?) && (linux_target? || (windows_target? && do_windows_sysprobe != ""))
+  sysprobe_support = (not heroku_target?) && (linux_target? || (windows_target? && do_windows_sysprobe != "")) &&
```

Comment:
Not related to your change, but interestingly a similar condition is defined here: https://github.com/DataDog/datadog-agent/blob/ca2705f9cfb8270a9707e96a0262c976ef469cd4/omnibus/config/projects/agent.rb#L225

Would it make sense to take this opportunity to refactor this by moving all the section under `if sysprobe_support` to the `system_probe` definition? That would drastically reduce the proliferation of these conditions that need to be repeated in several places.

Reply:
That seems like a fair suggestion, I'll do the change :)